devicetypes/smartthings/zigbee-metering-plug.src/zigbee-metering-plug.groovy

```diff
@@ -154,3 +164,14 @@ def configure() {
            zigbee.configureReporting(zigbee.SIMPLE_METERING_CLUSTER, ATTRIBUTE_READING_INFO_SET, DataType.UINT48, 1, 600, 1) +
            zigbee.electricMeasurementPowerConfig(1, 600, 1) 
 }
+
+def isOnOffStateChanged(Map event) {

```

Comment:
This is different than the change I proposed in the ticket. Since we're trying to avoid duplicate events, I don't think this implementation is going to achieve the desired effect state is not atomic nor does it provide read-after-write guarantees. Instead the change should be to ignore switch events that are created on receipt of a Default Response message. They you won't need to rely on any saved state.

Reply:
@tpmanley Sorry for late response. Due to remote work situtation I had to wait for a device. 

I have changed the implementation so that it does not relay on state. Instead, it just ignores the second event that was earlier created from "catchall" description. I've checked the event lists and "switch" is not doubled anymore. 

Could you please take a look at the code?